routes/like_dislike.route.js

```diff
@@ -0,0 +1,32 @@
+const express = require("express");
+const _ = require("lodash");
+const router = express.Router();
+const { checkUser } = require("../middleware/middlewares");
+const {
+  updateLikedVideo,
+  updateDisLikedVideo,
+} = require("../middleware/liked.middleware");
+router
+  .route("/:userId/like")
+  .get(checkUser, async (req, res) => {
+    const user = req.user;
+    const populateVideoData = await user.populate("likedvideos").execPopulate();
+    res.json({ success: true, likedvideos: populateVideoData.likedvideos });
```

Comment:
same for `likedvideos` & `disklikedvideos`  can be written as `likedVideos` & `dislikedVideos` just a suggestion.